triggering-jenkins.md

```diff
@@ -0,0 +1,93 @@
+# Triggering a Jenkins build as a task on XebiaLabs
+
+This guide will demonstrate how to set a XebiaLabs task to trigger a Jenkins build. 
+
+## Prerequisites
+
+* Jenkins is installed
+
+* Have an account with XebiaLabs
+
+#### 1.1 Create a new Jenkins job
+
+On the left side of the site, click on 'New Item' to create a new project. 
+
+![new-item](assets/new-item.png)
+
+Enter in a project name and select the project type 'Freestyle project'. 
+
+![new-project](assets/new-project.png)
+
+In the configuration settings, go the 'Build' section. Click on 'Add build step' and select 'Execute shell'. 
+
+![build-step](assets/build-step.png)
+
+Enter the necessary commands. The below example will execute the android-app-test script sample from Kobiton.
+
+```
+#!/bin/bash
+[ ! -d "samples" ] && git clone https://github.com/kobiton/samples.git || echo "Samples already exist"
+cd samples/javascript
+npm install
+npm run android-app-test
+```
+
+In your automation test, make sure to include your own Kobiton username and API key as well as the desired device and platform you wish to test with, which can be found on the Kobiton website.
+
+This guide is executing the NodeJS test script from the Kobiton samples, so we will add the NodeJS Plugin on Jenkins to execute NodeJS script as a build step. To do this, go to 'Manage Jenkins' -> 'Manage Plugins' -> Available. Search for NodeJS and install it.
```

Comment:
I completely get lost by this part, I don't understand the relevant between this part and the above parts ? Is it a separate part ? 

I would like to suggest the structure: 
```
# Triggering a Jenkins build as a task on XebiaLabs
## Prerequisites
....
##  Jenkins configuration 
### 1. Integrate NodeJs Plugin
....

### 2. Create a new Jenkins job
....

## XebiaLabs Configuration
### 1. Install the Jenkins plugin
...
### 2. Update Jenkins plugin settings
...

... so on
```

By the ways, NodeJS Plugin must be installed or just an optional for the sample above ?

Reply:
For this example that's executing a NodeJS script, the plugin is necessary. I will make it more clear.